pulp_deb/app/settings.py

```diff
@@ -6,3 +6,7 @@
 """
 
 FORBIDDEN_CHECKSUM_WARNINGS = True
+
+# Defines which publishers should be used for autopublish (along with which options) for
+# repositories with autopublish=True but no defined autopubish_modes.
+DEFAULT_AUTOPUBLISH_MODES = "simple structured verbatim"
```

Comment:
```suggestion
DEFAULT_AUTOPUBLISH_MODES = "structured"
```
The default should be structured only!

Perhaps we should actually limit the feature to structured only to begin with, and then see if there are any requests for other modes later.